concepts/stable-seekers/faq-stable.md

```diff
@@ -4,29 +4,35 @@ description: Common questions about getting and exchanging the Ppotocol's stable
 
 # FAQ - Stable Seekers and Holders
```

Comment:
Can add a question: what are the stablecoins of the protocol?

So far, just agEUR